tests/jvm/src/test/resources/lowlevel.expect

```diff
@@ -536,23 +558,8 @@ _root_.com.javacp.MetacJava#overload(A+1). => def overload: (a: <?>): Unit
 _root_.com.javacp.MetacJava#overload(A+2). => def overload: (a: <?>): Unit
   a => _root_.com.javacp.MetacJava#overload(A+2).(a)
   Unit => _root_.scala.Unit#
-_root_.com.javacp.MetacJava. => final object MetacJava
-_root_.com.javacp.MetacJava.StaticInner# => class StaticInner.{+4 decls}
-  extends Object
-_root_.com.javacp.MetacJava.StaticInner#NonStatic# => class NonStatic.{+2 decls}
-  extends Object
-_root_.com.javacp.MetacJava.StaticInner#NonStatic#`<init>`(). => primaryctor <init>: (): StaticInner.this.NonStatic
-  StaticInner => _root_.com.javacp.MetacJava.StaticInner#
-  NonStatic => _root_.com.javacp.MetacJava.StaticInner#NonStatic#
-_root_.com.javacp.MetacJava.StaticInner#NonStatic#method(NonStatic). => def method: (e: <?>): Unit
-  e => _root_.com.javacp.MetacJava.StaticInner#NonStatic#method(NonStatic).(e)
-  Unit => _root_.scala.Unit#
-_root_.com.javacp.MetacJava.StaticInner#`<init>`(). => primaryctor <init>: (): StaticInner
-  StaticInner => _root_.com.javacp.MetacJava.StaticInner#
-_root_.com.javacp.MetacJava.StaticInner#isNotStatic(). => def isNotStatic: (): Unit
-  Unit => _root_.scala.Unit#
-_root_.com.javacp.MetacJava.StaticInner. => final object StaticInner
-_root_.com.javacp.MetacJava.StaticInner.isStatic(). => def isStatic: (): Unit
+_root_.com.javacp.MetacJava#overload(A+3). => def overload: (a: <?>): Unit
```

Comment:
Why is this a `<?>`?

Reply:
Not sure, looking at lowlevel.expect there seems to be quite a few other `<?>` which seem unrelated to this PR. I opened https://github.com/scalameta/scalameta/issues/1418 to track fixing these. I don't think this is a blocker at the moment.